Backpack-SwiftUI/Calendar/Classes/Core/CalendarTypeContainerFactory.swift

```diff
@@ -18,23 +18,13 @@
 
 import SwiftUI
 
-public struct CalendarAccessibilityConfiguration {
```

Comment:
reverted this to the way it was handled before, reducing duplication and potential invalid states